local.properties.example

```diff
@@ -41,6 +41,10 @@ signing.keyId=
 signing.password=
 # Exported as e.g. `gpg --export-secret-keys > secring.gpg`
 signing.secretKeyRingFile=
+## When you want to use GnuPG command instead of exported keyring
+## uncommented and add key ID
+#signing.gnupg.keyName=
+#signing.gnupg.executable=gpg
```

Comment:
I think we should improve the comments to explain better what is required and used for what.

Are `signing.{keyId,password,secretKeyRingFile}` used at all when configured with `useGpgCmd()`?

Reply:
> Are signing.{keyId,password,secretKeyRingFile} used at all when configured with useGpgCmd()?

Yes.

These properties are documented at [gradle manual](https://docs.gradle.org/current/userguide/signing_plugin.html#sec:sec:gnupg_signatory_configuration)

Reply:
I've updated comments.